### ISSUE TITLE
[BUG] URL field of getBotCallbackAnswer is not processed

### ISSUE THREAD

**Issue by IRuslan** [morethanwords/tweb#274]
**Describe the bug**
Based on [official documentation](https://core.telegram.org/api/bots/buttons#callback-queries) of the keyboardButtonCallback, once a button is pressed by the user, the getBotCallbackAnswer method should be called. And there is a expected response of messages.botCallbackAnswer type. One of options is URL to the bot itself via URL field. However WebK version of Telegram client abandon this option while the same configuration works correctly on iOS and Desktop application.

**To Reproduce**
Steps to reproduce the behavior (here we assume you have a own bot to interact with Telegram client):
1. Use a bot to send inlineKeyboard with keyboardButtonCallback.
2. Click on the button
3. Bot receives callback query
4. Bot sends a response in a form of messages.botCallbackAnswer with `url` and `has_url` fields accordingly to [documentation](https://core.telegram.org/api/bots/buttons#sending-a-callback-query)
5. Nothing happens on the client side

**Expected behavior**
Telegram client should switch the user to specified URL once messages.botCallbackAnswer received.

**Desktop (please complete the following information):**
 - Linux
 - Chrome
 - Version 113

**Additional context**
After quick overview, I have found a [WebK source code](https://github.com/morethanwords/tweb/blob/dc2769849a7d05f679a2906de2fdb1e20b046767/src/components/chat/bubbles.ts#L4291) responsible for processing of `message` field of the botCallbackAnswer. However there is nothing similar about `url` field.

**Comment by morethanwords** [comment]
This will be implemented together with games.

**Comment by IRuslan** [comment]
I wonder if there is any roadmap or issue to track?

Also, just a feeling that this is a part related to bots functionality and partially to games. While other features related to bots working maybe it makes sense to treat it separately from games (sounds like a separate huge feature request).

**Comment by morethanwords** [comment]
> Just as idea what need to be changed on line 4291 of buubles.ts.

It's not that simple, see [this](https://github.com/Ajaxy/telegram-tt/blob/master/src/global/actions/api/messages.ts#L1463). What do you need it for?

**Comment by IRuslan** [comment]
> It's not that simple, see [this](https://github.com/Ajaxy/telegram-tt/blob/master/src/global/actions/api/messages.ts#L1463).

Agree. I just was not aware if any similar implementation already exists in WebK version.

> What do you need it for?

I want to build the following workflow for the bot:
1) Bot send a message with inline button to Telegram group
2) Telegram user presses the button
3) Bot receives CallbackQuery event to provide initial processing
4) Then based on results bot sets up a callbackAnswer to redirect user to DM with bot (Telegram API allows only this kind of URLs for callbackAnswer) to continue interaction.
